topics/transcriptomics/tutorials/scrna-seq-basic-pipeline/tutorial.md

```diff
@@ -0,0 +1,910 @@
+- Analysis is more subjective than we think, and biological understanding of the samples as well as many iterations of analysis are important to give us our best change of attaining real biological insights
+requirements:
+-
+    type: "internal"
+    topic_name: transcriptomics
+    tutorials:
+        - droplet-quantification-processing
+tags:
+- single-cell
+- 10x
+contributors:
+- nomadscientist
+
+---
+
+
+# Introduction
+{:.no_toc}
+
+<!-- This is a comment. -->
+
+You've done all the work to make a single cell matrix, with gene counts and mitochondrial counts and buckets of cell metadata from all your variables of interest (or, if not, please see [this tutorial](https://training.galaxyproject.org/training-material/topics/transcriptomics/tutorials/droplet-quantification-preprocessing/tutorial.html) to do so!) Now it's time to fully process our data, to remove low quality cells, to reduce the many dimensions of data that make it difficult to work with, and ultimately to try to define our clusters and to find our biological meaning and insights! There are many packages for analysing single cell data - Seurat, Scanpy, Monocle3, Scater, and so forth. We're working with Scanpy, because currently Galaxy hosts the most Scanpy tools of all of those options.
+
+> ### {% icon comment %} Tutorials everywhere?
+> This tutorial is similar to another fantastic tutorial: [Clustering 3k PBMC](
+https://training.galaxyproject.org/training-material/topics/transcriptomics/tutorials/scrna-scanpy-pbmc3k/tutorial.html). That tutorial will go into much further depth on the analysis, in particular the visualisation and science behind identifying marker genes. Their experimental data is clean and well annotated, which illustrates the steps beautifully. Here, we work more as a case study with messier data, to help empower you in making choices during the analysis. We highly recommend you work through all the galaxy single cell tutorials to build confidence and expertise! For trainers, note that there are small-group options in this tutorial.
+{: .comment}
+
+> ### Agenda
+>
+> In this tutorial, we will cover:
+>
+> 1. TOC
+> {:toc}
+>
+{: .agenda}
+
+## Get data
+
+We've provided you with experimental data to analyse from a mouse dataset of fetal growth restriction {% cite Bacon2018 %}. This is the full dataset generated from [this tutorial](https://training.galaxyproject.org/training-material/topics/transcriptomics/tutorials/droplet-quantification-preprocessing/tutorial.html) if you used the full fastq files rather than the subsampled ones (see the study in Single Cell Expression Atlas [here](https://www.ebi.ac.uk/gxa/sc/experiments/E-MTAB-6945/results/tsne) and the project submission [here](https://www.ebi.ac.uk/arrayexpress/experiments/E-MTAB-6945/)). You can find this dataset in this [input history](https://humancellatlas.usegalaxy.eu/u/wendi.bacon.training/h/filter-plot-and-explore-single-cell-rna-seq-data---input) or download from Zenodo below.
```

Comment:
fastq → FASTQ

Reply:
@mtekman are you familiar with github's suggestion mode? It'd save @nomadscientist time when she goes to apply your review comments

Reply:
oh whoops, I forgot about that...

Reply:
went ahead and edited your feedback to do it :)